core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
nit: Did we mean to remove this line?

Reply:
Yeah I did, usually there isn't a new line after the license stuff right

Reply:
I think some files have it and some don't.